CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 CHANGELOG for Sulu
 ==================
 
+* master
```

Comment:
That would actually be `release/1.6` :slightly_smiling_face:

Reply:
fixed